beets/ui/commands.py

```diff
@@ -1364,6 +1359,21 @@ def modify_items(lib, mods, dels, query, write, move, album, confirm):
             obj.try_sync(write, move)
 
 
+def print_modify_item(obj, mods, dels):
```

Comment:
More naming stuff: I suggest `print_and_modify`, without item, because the argument can either be an Item or an Album.